utils/getBuyAssetExtrinsic.ts

```diff
@@ -0,0 +1,22 @@
+import { CENNZAsset } from "@/types";
+import { Api } from "@cennznet/api";
+import { SubmittableExtrinsic } from "@cennznet/api/types";
+import { Balance } from "@/utils";
+
+// TODO: Need test
+export default function getBuyAssetExtrinsic(
+	api: Api,
+	exchangeAssetId: CENNZAsset["assetId"],
+	exchangeAssetValue: Balance,
+	receiveAssetId: CENNZAsset["assetId"],
+	receivedAssetValue: Balance,
+	slippage: number
+): SubmittableExtrinsic<"promise"> {
+	return api.tx.cennzx.buyAsset(
+		null,
```

Comment:
do we have advance setting where exchange goes to a third account from UI in which case the first param won't be null

Reply:
no there isn't atm, would be nice. Created an issue to track it https://github.com/cennznet/app-hub/issues/142